File: chat/v2/credential/create-credentials/create-credentials.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

credential = @client.chat.credentials.create(
                                        api_key: 'apiKey',
                                        friendly_name: 'Friendly Name',
                                        type: 'gcm'
                                      )

puts credential.sid

File: chat/v2/credential/create-credentials/create-credentials.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$credential = $twilio->chat->v2->credentials
                               ->create("gcm", // type
                                        array(
                                            "apiKey" => "apiKey",
                                            "friendlyName" => "Friendly Name"
                                        )
                               );

print($credential->sid);